update-server-repo-setup/generate-repos-config

```diff
@@ -157,41 +157,42 @@ class RepoConfigGenerator:
                     msg += 'by SCC. Skipping product.'
                     print(msg % triplet)
                     continue
-                
+
                 for repo in scc_product:
                     fs_location = repo['url'].split('suse.com')[-1]
                     repo_desc = repo['description']
-                    if (repo['enabled']):
-                        if self._add_repo(repo_desc, exclusions):
```

Comment:
Reversing the order adds an extra function call for all repos that we can see in SCC but are not enabled for our subscription. That is not OK.